main.tf

```
@@ -3,13 +3,14 @@
 # ------
 locals {
   uptime_monitoring_display_name = var.uptime_monitoring_display_name != "" ? var.uptime_monitoring_display_name : var.uptime_monitoring_host
+  exist_path                     = var.uptime_monitoring_path != "" || var.uptime_monitoring_path != "/" ? false : true
```

I think that this check is not correct: if the path is not empty (first check) the result will be `false`.

I would also simplify the syntax:

```suggestion
  exist_path                     = var.uptime_monitoring_path != "" && var.uptime_monitoring_path != "/"
```